### PR TITLE
PS-1062: Create post-visit sharing API

### PR DESCRIPTION
**Pull request by andyartz**
- Clinical notes updates
- Standardized language across requests
- HTTP 403 Forbidden responses 
- Some structural updates based on latest OpenApi build

### CODE REVIEW

File: patient-appointment-api.yml
Comment (jjathman-ch) [inline patient-appointment-api.yml:1]: I think the version should be upped to 1.5.0 since this is a new API.

File: patient-appointment-api.yml [hunk 1]
```diff
@@ -903,6 +969,69 @@ components:
...
+        activeSnomedDiagnoses:
+          type: array
+          description: All active diagnoses (problems) known about the patient. Must
+            all be SNOMED codes. If the patient is an existing patient then this list
+            should be populated with all known diagnoses. **ICD-10 should be greatly
+            preferred if available.**
+          example:
+            - code: "34713006"
+              displayName: Vitamin D deficiency
+              date: 2023-11-12
```
Comment (jjathman-ch) [inline patient-appointment-api.yml:1014]: I think all dates like this need to be manually quoted or it doesn't look correct when deployed to Mintlify. Could test it out but I think that's still the case. I can't get MN to do the quoting for us.

**Review by jjathman-ch — APPROVED** [review]
Couple things to review.

File: patient-appointment-api.yml (hunk 1 above)
Comment (andyartz) [reply patient-appointment-api.yml:1014]: Argh, missed this one, thanks

Comment (andyartz) [reply patient-appointment-api.yml]: two, actually. done!

File: patient-appointment-api.yml
Comment (andyartz) [reply patient-appointment-api.yml:1]: done!

**Pull request merged**